boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -41,6 +42,7 @@ FAKE_VALUE_FUNC(float, get_left_wheel_speed);
 FAKE_VALUE_FUNC(float, get_right_wheel_speed);
 FAKE_VALUE_FUNC(float, get_steering_angle);
 FAKE_VALUE_FUNC(float, get_brake_pressure);
+FAKE_VALUE_FUNC(bool, get_brake_actuation_status);
```

Comment:
This should be renamed to 

`is_brake_actuated`  because it's returning a boolean. Naming something with `status` makes it vague.

Reply:
fixed

Reply:
As a general note it should match the name of the corresponding Io function (i.e. `Io_XXXX_IsBrakeActuated`